src/pages/examples.tsx

```diff
@@ -274,6 +276,9 @@ export default function Examples() {
 								.use(Unsplash, { companionUrl })
 								.use(Box, { companionUrl });
 
+							// Expose for easier debugging
+							globalThis.uppy = uppy;
```

Comment:
You need to place this in a `useEffect` because you only want this to happen once, not every state change.

Reply:
But Uppy also gets recreated on every state change, so an effect wouldn’t make a difference. I think as long as uppy is recreated on every render, this is the correct way to do it.